modules/video/src/lkpyramid.hpp

```diff
@@ -7,6 +7,15 @@ namespace detail
 
     typedef short deriv_type;
 
+    struct SharrDerivInvoker : ParallelLoopBody
```

Comment:
This implementation is not mine - it was here, and was merely parallelized. Do you want me to use Scharr() instead?

Reply:
Looks like related changes are pretty small, so we can keep the current state.

BTW, Instead of `Mat*` we can use `mutable Mat&` in `ParallelLoopBody` classes (operator() is "const").